tutor/resources/styles/components/notes/summary-page.scss

```diff
@@ -1,7 +1,59 @@
-.overlay.notes-summary {
-  background-color: $tutor-neutral-light;
+.modal.show .modal-dialog.notes-modal {
+  // Force fixed modal-scroll-top button to attach to modal
+  // by using a no-effect transform.
+  transform: rotate(0);
+}
+
+.modal-dialog.notes-modal {
+  margin: 40px auto 24px;
+  height: calc(100% - 64px);
+  width: calc(100% - 48px);
+  max-width: $tutor-max-panel-width;
   justify-content: center;
 
+  .modal-header {
+    background-color: $tutor-dark-blue;
+
+    &, .close {
+      color: $tutor-white;
+    }
+
+    .close {
+      opacity: 1;
+      text-shadow: none;
+      font-size: 2.75rem;
+      margin-top: -1.75rem;
+    }
+  }
+
+  .modal-content {
+    // Fix white background showing behind rounded header corners
+    background: transparent;
+  }
+
+  .modal-body {
+    padding: 0;
+    background-color: $tutor-neutral-light;
+  }
+
+  .modal-scroll-top {
+    position: fixed;
+    bottom: 44px;
+    right: 44px;
+    padding: 4px;
+    text-align: center;
+    font-size: 16px;
+
```

Comment:
We use rems for all font sizes, and 1 rem is set to 10px.  This should be: `font-size: 1.6rem;`

Also it'd be nice to set the `44px` to a variable at the top of the file, perhaps `$notes-modal-page-border-size` or something like that and use that everywhere so that it could be easily tweaked.

Reply:
Whoops, missed this ref while trying to make it match the spec. Good catch.